log4j-api/src/test/java/org/apache/logging/log4j/util/LoaderUtilTest.java

```diff
@@ -55,4 +63,53 @@ public void systemClassLoader() {
             thread.setContextClassLoader(tccl);
         }
     }
+
+    @Test
+    public void securityManager() {
```

Comment:
This should also add the annotation `@ResourceLock("java.lang.SecurityManager")` as categorized in https://issues.apache.org/jira/browse/LOG4J2-2921